### DISCUSSION TITLE
"error": "Ordering by -started is not allowed"

### DISCUSSION THREAD

**Discussion by pe5ha** [online-go/online-go.com#2377] [Q&A] [answered]
Hello. I have a program that used such API requests: 
https://online-go.com/api/v1/players512029/games?ordering=-started&ended__isnull=false&page=1&page_size=100

It worked well before. But recently, these requests have stopped working. "error": "Ordering by -started is not allowed" returns.
Has something changed in the API? How can I sort games by start date now?

**Comment by anoek** [comment, marked as answer]
We had to lock down that endpoint a bit as it was being abused by some miscreants. 

Will `https://online-go.com/api/v1/players512029/games?ordering=-id&ended__isnull=false&page=1&page_size=100`

work for you instead of `-started` ? It should provide the same ordering but it's a bit easier on the database.

**Reply by pe5ha** [reply]
Thank you!
